### DISCUSSION TITLE
Key constraint on domains

### DISCUSSION THREAD

**Discussion by neri-ripio** [WalletConnect/walletconnect-monorepo#2399] [Developer Support]
Hello!, we have more than one project (mobile and extension wallet) using the same domain, and we are getting an error on the domain validation, could this be related to that reason? and how can we validate the projects domains?

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/129459236/9c77f64c-82f0-4c9c-8d31-b34182e92dcb)

**Comment by finessevanes** [comment]
Your discussion has been viewed and is getting investigated.

**Comment by Cali93** [comment]
@neri-ripio Thanks for reporting the issue and your patience.
We fixed it on the latest release.
